<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1453</width>
    <height>40</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1453</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1453</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="PyDMLabel" name="device_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="channel" stdset="0">
      <string>ca://${P}${ARBITER}:AP:Entry:${POOL}:Device_RBV</string>
     </property>
     <property name="displayFormat" stdset="0">
      <enum>PyDMLabel::String</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMLabel" name="id_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>50</number>
     </property>
     <property name="channel" stdset="0">
      <string>ca://${P}${ARBITER}:AP:Entry:${POOL}:ID_RBV</string>
     </property>
     <property name="displayFormat" stdset="0">
      <enum>PyDMLabel::Decimal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="rate_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color : black; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMLabel" name="beamclass_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>loc://${P}${ARBITER}${POOL}:MaxBC?type=int&amp;init=0</string>
     </property>
     <property name="showUnits" stdset="0">
      <bool>true</bool>
     </property>
     <property name="channel" stdset="0">
      <string>loc://${P}${ARBITER}${POOL}:MaxBC?type=int&amp;init=0</string>
     </property>
     <property name="displayFormat" stdset="0">
      <enum>PyDMLabel::Default</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BCByteIndicator" name="beamclass_bytes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>183</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>183</width>
       <height>16</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="channel" stdset="0">
      <string>ca://${P}${ARBITER}:AP:Entry:${POOL}:BeamClassRanges_RBV</string>
     </property>
     <property name="onColor" stdset="0">
      <color>
       <red>21</red>
       <green>165</green>
       <blue>62</blue>
      </color>
     </property>
     <property name="orientation" stdset="0">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="showLabels" stdset="0">
      <bool>false</bool>
     </property>
     <property name="bigEndian" stdset="0">
      <bool>true</bool>
     </property>
     <property name="circles" stdset="0">
      <bool>false</bool>
     </property>
     <property name="numBits" stdset="0">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMLabel" name="transmission_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="showUnits" stdset="0">
      <bool>true</bool>
     </property>
     <property name="channel" stdset="0">
      <string>ca://${P}${ARBITER}:AP:Entry:${POOL}:Transmission_RBV</string>
     </property>
     <property name="displayFormat" stdset="0">
      <enum>PyDMLabel::Exponential</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EvByteIndicator" name="energy_bytes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="channel" stdset="0">
      <string>ca://${P}${ARBITER}:AP:Entry:${POOL}:eVRanges_RBV</string>
     </property>
     <property name="onColor" stdset="0">
      <color>
       <red>21</red>
       <green>165</green>
       <blue>62</blue>
      </color>
     </property>
     <property name="orientation" stdset="0">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="showLabels" stdset="0">
      <bool>false</bool>
     </property>
     <property name="bigEndian" stdset="0">
      <bool>true</bool>
     </property>
     <property name="circles" stdset="0">
      <bool>false</bool>
     </property>
     <property name="numBits" stdset="0">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMLabel" name="cohort_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="channel" stdset="0">
      <string>ca://${P}${ARBITER}:AP:Entry:${POOL}:Cohort_RBV</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMByteIndicator" name="live_byte">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="channel" stdset="0">
      <string>ca://${P}${ARBITER}:AP:Entry:${POOL}:Live_RBV</string>
     </property>
     <property name="showLabels" stdset="0">
      <bool>false</bool>
     </property>
     <property name="circles" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMByteIndicator</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.byte</header>
  </customwidget>
  <customwidget>
   <class>BCByteIndicator</class>
   <extends>PyDMByteIndicator</extends>
   <header>widgets</header>
  </customwidget>
  <customwidget>
   <class>EvByteIndicator</class>
   <extends>PyDMByteIndicator</extends>
   <header>widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
